recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set version = "4.5.4" %}
-{% set use_n_python_patch = os.environ.get('OPENCV_USE_N_PYTHON_PATCH', '0') %}
```

Comment:
I don't know what this is about but is there maybe a patch file coming with this that needs to be removed or is this just cruft?